src/types.tsx

```diff
@@ -222,6 +232,37 @@ export interface ScreenProps extends ViewProps {
    * - "landscape_right" – landscape-right orientation is permitted
    */
   screenOrientation?: ScreenOrientationTypes;
+  /**
+   * Describes heights where a sheet can rest.
+   * Works only when `stackPresentation` is set to `formSheet`.
+   * Defaults to `large`.
+   *
+   * @platform ios
+   */
+  sheetAllowedDetents?: SheetDetentTypes;
```

Comment:
And available options in this file too.

Reply:
[bcfbc09](https://github.com/software-mansion/react-native-screens/pull/1649/commits/bcfbc093d1237092400b6e715c3967e5d11d6e08)